### ISSUE TITLE
Compatibility with Docker Compose 1.23.0+

### ISSUE THREAD

**Issue by spitlo** [philipnewcomer/wpdocker#115]
Since 1.23.0 ([release notes](https://github.com/docker/compose/releases/tag/1.23.0)), Docker Compose adds a hash at the end of container names, breaking Pilothouse functionality (more specifically `getContainerInternalIp`).

I was able to fix this locally by getting container name from `docker ps`, but I'm not sure that is the best way to go about it.

Anyhow, my local, working version of `getContainerInternalIp` looks like this:
```JavaScript
function getContainerInternalIp(container) {
	const containerPrefix = `pilothouse_${container}_`
	const containerName = commands.shellCommand(environment.runDirectory, 'docker', [
		'ps',
		'-a',
		'--format', '{{.Names}}',
		'--filter', `name=${containerPrefix}`,
	], true);
	return commands.shellCommand(environment.runDirectory, 'docker', [
		'inspect',
		'--format', '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}',
		containerName,
	], true);
}
```

**Comment by philipnewcomer** [comment]
Thanks for the heads up @spitlo. I've released `0.14.4` which should resolve this issue.

**Comment by spitlo** [comment]
Works perfectly, thanks!
